src/overlays/gamestates/ovl_title/z_title.c

```diff
@@ -91,7 +91,7 @@ void Title_Draw(GameState* thisx) {
 
     gDPPipeSync(POLY_OPA_DISP++);
     gDPSetCycleType(POLY_OPA_DISP++, G_CYC_2CYCLE);
-    gDPSetRenderMode(POLY_OPA_DISP++, G_RM_XLU_SURF2, G_RM_OPA_CI | CVG_DST_WRAP);
+    gDPSetRenderMode(POLY_OPA_DISP++, G_RM_PASS, G_RM_XLU_SURF2 | CVG_DST_WRAP);
```

Comment:
```suggestion
    gDPSetRenderMode(POLY_OPA_DISP++, G_RM_PASS, G_RM_CLD_SURF2);
```
( https://github.com/zeldaret/oot/pull/1313#discussion_r918950375 )